code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -66,7 +68,14 @@
 
 /obj/structure/closet/initialize()
 	..()
-	spawn_contents()
+	if (!time_initialized_at)
+		time_initialized_at = world.time
+		spawn_contents()
+	else
+		//haha, so you'd like to initialize twice huh? you got some explaining to do kid.
```

Comment:
Make this more aggressive

Reply:
I originally wrote "gotchu bitch"

Reply:
You don't understand, you have to make it sound a lot more like someone who is smug about it. Someone whose style of speech is so aggravating that you want to cave their face in with the heel of your boot, something that the map-bot guy would say, something that gets under the skin like the "wholesome" users that are actually incredibly toxic in rule-heavy places, something that is proceeded with heavy smile emoji usage, something like that.

Reply:
>something that the map-bot guy would say

kek